.ci/update-archetypes.sh

```diff
@@ -16,5 +16,5 @@ for f in ./client-archetype-* ; do
   fi
 done
 
-git -c user.name="Gluon Bot" -c user.email="githubbot@gluonhq.com" commit */$XML_LOCATION -m "Upgrade client-maven-plugin version to $1"
+git commit */$XML_LOCATION -m "Update client-maven-plugin version to $1"
```

Comment:
I assume the github `user.name` and `user.email` variables are set globally and are persisted during the entire travis job. These scripts (`update-archetypes.sh` and `update-samples.sh`) are never called apart from `release.sh`?

Reply:
No, they are not. Updating samples and archetypes are a part of the release process and only called from release script.